src/transformers/models/clvp/modeling_clvp.py

```diff
@@ -1511,11 +1515,6 @@ def fix_speech_decoder_output(self, speech_ids: torch.LongTensor) -> torch.LongT
                 This refers to the output of the decoder model.
         """
         decoder_fixing_codes = self.config.decoder_config.decoder_fixing_codes
-        speech_ids = speech_ids[:, 1:]
```

Comment:
We can't remove this here - it would change the output of the current model when doing a forward pass